src/main/java/core/basesyntax/service/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriter {
+    public void write(String text, String filePath);
```

Comment:
According to Java conventions, the 'public' modifier is redundant in interface method declarations, as all methods in an interface are public by default.